win32/perllib.c

```diff
@@ -67,6 +68,13 @@ win32_checkTLS(PerlInterpreter *host_perl)
         }
         host_perl->Isys_intern.cur_tid = tid;
     }
+#else
+    dTHX;
+    if (host_perl != my_perl) {
+        int *nowhere = NULL;
```

Comment:
Why this assignment?

Reply:
An unthinking C&P from the original, there was a `*nowhere = 0;` removed in 7bd379e81.

I'll remove that last vestige from both sides of the if().